src/scripts/routes.js

```diff
@@ -308,12 +308,6 @@ import { appRouter } from '../components/appRouter';
         type: 'home'
```

Comment:
This file should probably be moved to `routes/` and renamed. Not sure if that would best fit in this PR or a follow up.